app/src/release/java/org/mifos/mobilebanking/api/SelfServiceInterceptor.java

```diff
@@ -20,20 +20,22 @@
  * @since 21/06/16
  */
 public class SelfServiceInterceptor implements Interceptor {
-
     public static final String HEADER_TENANT = "Fineract-Platform-TenantId";
     public static final String HEADER_AUTH = "Authorization";
+    public static final String DEFAULT_TENANT = "default";
```

Comment:
are we using anywhere this field?

Reply:
This will be used in the release flavour of the app as you might remember that earlier we set default values for debug flavour and for release flavour.